.github/workflows/admin.yml

```diff
@@ -0,0 +1,74 @@
+name: Admin
+on:
+  push:
+    branches:
+      - trunk
+  pull_request:
+  # TODO: paths: src/Resources/app/administration
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: octo-sts/action@main
```

Comment:
I think we should move the common setup part into a local action (I believe we've also done this in rufus?). That make it more readable and DRY